include/SFML/Network/Ftp.hpp

```diff
@@ -484,7 +484,7 @@ class SFML_NETWORK_API Ftp : NonCopyable
     /// \see download
     ///
     ////////////////////////////////////////////////////////////
-    Response upload(const std::string& localFile, const std::string& remotePath, TransferMode mode = Binary);
+    Response upload(const std::string& localFile, const std::string& remotePath, TransferMode mode = Binary, const append = false);
```

Comment:
They added a new "append" type and implicit parameter names in C++16, didn't ya hear?  ( ͡° ͜ʖ ͡° )